adapter/internal/api/models/error_list_item.go

```diff
@@ -1,6 +1,6 @@
 // Code generated by go-swagger; DO NOT EDIT.
 
-// Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+// Copyright (c) 2021, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
2020?

Reply:
these were changed when auto gen code, will revert it

Reply:
fixed with https://github.com/wso2/product-microgateway/pull/1826/commits/3aa9b7de891d72fbd25fdfb61f7dac23d0f11d99